build.sbt

```diff
@@ -6,15 +6,15 @@ import com.lucidchart.sbt.scalafmt.ScalafmtCorePlugin.autoImport._
 ThisBuild / organization := "com.gu"
 ThisBuild / name := "table-versions"
 ThisBuild / description := "Version control for your Big Data!"
-
+lazy val scalatest = "org.scalatest" %% "scalatest" % "3.0.5"
```

Comment:
Just remembered: this project has a file `Dependencies.scala` so it probably makes sense to put shared dependencies there.